lib/rules/web/admin_console/4.4/user_management.xyaml

```diff
@@ -33,3 +33,26 @@ click_stop_impersonation:
   element:
     selector: *stop_impersonation
     op: click
+check_data_in_users_table:
```

Comment:
Move rules in `users.xyaml` here so we can reduce one file

Reply:
If change the rules location, seems we also need update them for 4.3 rule?

Reply:
We can update for 4.3, also it seems harmless if don't change to 4.3, right?